authentik/blueprints/apps.py

```diff
@@ -13,21 +13,23 @@ class ManagedAppConfig(AppConfig):
 
     _logger: BoundLogger
 
+    RECONCILE_PREFIX: str = "reconcile_"
```

Comment:
Now looking at it again I think there's probably a better way to do this than using `dir()` and then `getattr()`, probably something using a decorator?

Pseudo code:

```
@self.reconcile
def do_something(self):
    pass
```

(Not really part of this PR tbh)

Reply:
Yeah, it would be a lot better than filtering on function prefixes x)

Reply:
Created https://github.com/goauthentik/authentik/issues/7852 for it